.github/workflows/github-actions-generate-docs.yml

```
@@ -0,0 +1,34 @@
+name: Build docs
+on: [ push, workflow_dispatch ]
+
+jobs:
+  Explore-GitHub-Actions:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 18
```

```suggestion
          node-version: 16
```
16 is a stable release, would choose that.